src/test/java/com/artipie/management/FakeRepoPerms.java

```diff
@@ -40,36 +46,63 @@
 public final class FakeRepoPerms implements RepoPermissions {
 
     /**
-     * Repositories list.
+     * Permissions.
      */
-    private final List<String> repos;
+    private final Map<String, Pair<Collection<PermissionItem>, Collection<PathPattern>>> repos;
```

Comment:
@olenagerasimova type `Map<String, Pair<Collection<PermissionItem>, Collection<PathPattern>>>` is too hard to comprehend, I'd suggest to create private class instead of `Pair<Collection<PermissionItem>, Collection<PathPattern>>`